src/antsibull_docs_parser/parser.py

```diff
@@ -165,10 +169,19 @@ def _parse_option_like(
         text = text[len(_IGNORE_MARKER) :]
     else:
         plugin_identifier = context.current_plugin
+        entrypoint = context.role_entrypoint
+    if plugin_identifier is not None and plugin_identifier.type == "role":
+        idx = text.find(":")
+        if idx >= 0:
+            entrypoint = text[:idx]
+            text = text[idx + 1 :]
```

Comment:
Can you use `.partition()` here? I think it's a bit cleaner, but that's up to you.

```suggestion
    if plugin_identifier is not None and plugin_identifier.type == "role":
        part1, sep, part2 = text.partition(":")
        if sep:
            entrypoint = part1
            text = part2
```